borg/remote.py

```diff
@@ -166,7 +166,7 @@ def __init__(self, location, create=False, exclusive=False, lock_wait=None, lock
         self.preload_ids = []
         self.msgid = 0
         self.to_send = b''
-        self.cache = {}
+        self.get_cache = {}
```

Comment:
how about, instead of renaming the attribute, just put a comment to the right of it, explaining that this cache is only used for get?